util/types/etc.go

```diff
@@ -55,6 +55,11 @@ func IsTypeJSON(tp byte) bool {
 	return tp == mysql.TypeJSON
 }
 
+// IsTypeParam returns a boolean indicating whether the tp is the Unspecified type.
+func IsTypeParam(tp byte) bool {
```

Comment:
how about `s/IsTypeParam/IsTypeUnspecified/` ?

Reply:
Thank you for the suggestion. I'll change it.